xbmc/settings/Settings.cpp

```diff
@@ -418,6 +418,7 @@ const std::string CSettings::SETTING_SOURCE_VIDEOS = "source.videos";
 const std::string CSettings::SETTING_SOURCE_MUSIC = "source.music";
 const std::string CSettings::SETTING_SOURCE_PICTURES = "source.pictures";
 
+const std::string CSettings::SETTING_VIDEOSCREEN_SCALING_FILTER="videoscreen.scalingfilter";
```

Comment:
This can exist in the file it's used (`WinSystemGbm.cpp`) we do this to avoid polluting Settings.cpp with platform specifics